draft-ietf-tls-esni.md

```diff
@@ -383,21 +386,21 @@ connection negotiated this extension with the "ech_accept" response type. If so,
 this is referred to as an "ECH PSK". Otherwise, it is a "non-ECH PSK". This may
 be implemented by adding a new field to client and server session states.
 
-## Incorporating Outer Extensions {#outer-extensions}
+# The "ech_outer_extensions" Extension {#outer-extensions}
 
 Some TLS 1.3 extensions can be quite large and having them both in the inner and
 outer ClientHello will lead to a very large overall size. One particularly
 pathological example is "key_share" with post-quantum algorithms. In order to
 reduce the impact of duplicated extensions, the client may use the
-"outer_extensions" extension.
+"ech_outer_extensions" extension.
 
 ~~~
     enum {
        outer_extension(0xff04), (65535)
     } ExtensionType;
 
     struct {
-       ExtensionType outer_extensions<2..254>;
+       ExtensionType ech_outer_extensions<2..254>;
```

Comment:
Did you mean to rename the `ExtensionType` value instead?

Reply:
Yes, thanks for spotting this!

I apologize for that careless mistake.